packages/playground/fixtures/rspack.ts

```diff
@@ -28,7 +30,8 @@ class Rspack {
 				item();
 			}
 		});
-		this.devServer = new RspackDevServer(
+		const DevServerConstructor = wds ? WebpackDevServer : RspackDevServer;
```

Comment:
Presumably speaking, this might not be stable unless we apply the wds patch(packages/rspack-dev-server/src/patch.ts).
But here is working as expected, since we have already applied it.